core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -6,8 +6,7 @@
 
 {% block header_js %}
   {{ super() }}
-  {% if exploration_version %}
-    <script type="text/javascript">
+    <script ng-if="explorationVersion" type="text/javascript">
```

Comment:
@supra08, have you tested this page locally? is anywhere $scope.explorationVersion is defined? 
I think you should take a look at #4209 or other similar PRs (this can may help you understand).

Reply:
Sorry about that, I'll look into it and fix this as soon as possible

Reply:
Actually the problem I am facing is that I cannot find the relevant file in which to define $scope.explorationVersion as GLOBAL. I've been stuck in this part for some time. I would be highly glad if you could help me out with that. @DubeySandeep